liblangutil/Exceptions.h

```diff
@@ -189,6 +191,19 @@ class Error: virtual public util::Exception
 		return "Error";
 	}
 
+    static std::optional<Severity> stringToSeverity(std::string _severity)
+	{
+        _severity = tolower(_severity);
+
+        _severity = std::regex_replace(_severity, std::regex("^ +| +$|( ) +"), "$1");
```

Comment:
Using a regex for this is an overkill. Use `boost::trim_copy()`.

Reply:
Done 👍